samples/gcp-storage/.gitignore

```diff
@@ -0,0 +1,11 @@
+# package directories
+node_modules
+
+# Serverless directories
+.serverless
+
+coverage
+
+src/multicloud-gcp
+
+.gcloud
```

Comment:
It seems that this exclusion is not working, could you remove it? (or maybe fix it if it´s necessary)